ivy/container/sorting.py

```diff
@@ -59,6 +59,58 @@ def argsort(
             map_sequences=map_sequences,
             out=out,
         )
+    
```

Comment:
There shouldn't be a blank line here.

Reply:
Also, the docstring should be positioned properly similar to what I've suggested for the `array/sorting.py` file

Reply:
You also need to add a docstring to the `static_argsort` function.